### PR TITLE
Created Card Component

### PR DESCRIPTION
**Pull request by khyatibaria**
### Created Card Component
Please Review it
Working Link :- https://deploy-preview-11--pixel-u-i.netlify.app/card/card

### CODE REVIEW

**Review by nyyshaaa — APPROVED** [review]
Looks good to me, you forgot to make the cross button cursor to pointer.

File: card/card.css [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+  margin: 0.75rem 0.25rem;
+  padding: 0.25rem;
+}
+.card-footer {
+  align-self: center;
+}
+.card-button {
+  display: inline-block;
+  cursor: pointer;
+  background-color: var(--white);
```
Comment (bharati-21) [inline card/card.css:88]: A suggestion - please avoid using variable values as variable names.

File: card/variables.css [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  --p-8: 2rem;
+  --m-8: 2rem;
+  --fontSize-0: 0.5rem;
+  --fontSize-1: 1rem;
+  --fontSize-2: 1.25rem;
+  --fontSize-3: 1.5rem;
+  --fontSize-4: 1.75rem;
+  --fontSize-5: 2rem;
+  --radius: 0.25rem;
+}
```
Comment (bharati-21) [inline card/variables.css:63]: Why not add these values common to all the components? That is, say make one variables.css for all the components?

Are these variables and resets only specific to this component?

**Review by bharati-21 — APPROVED** [review]
Looks good to be merged!

**Pull request merged**